IS0401Test.py

```diff
@@ -1206,6 +1206,76 @@ def test_21(self, test):
 
         return test.PASS()
 
+    def test_22(self, test):
+        """Node resource IDs persist over a reboot"""
+
+        return test.MANUAL("This check must be performed manually, or via use of the following tool",
+                           "https://github.com/AMWA-TV/nmos-testing/blob/master/utilities/uuid-checker/README.md")
+
+    def test_23(self, test):
+        """Senders and Receivers correctly use BCP-002-01 grouping syntax"""
+
+        found_groups = False

```

Comment:
Should we distinguish `found_groups` from `found_senders_receivers` - the latter also being `False` probably ought to be reported as `UNCLEAR` "No Sender/Receiver resources were found on the Node."?

Reply:
I'll add this in. It does seem similar to the below, but might protect against a misconfiguration. Technically it will prevent the other noted issue giving an OPTIONAL too, perhaps further highlighting the error which will come from an earlier test.